src/GraphQL/Server.hs

```diff
@@ -37,18 +38,16 @@ import GraphQL.Internal.Schema (HasName(..))
 import GraphQL.API
 import GraphQL.Internal.Input (CanonicalQuery)
 
-import Control.Monad.Catch (MonadThrow, throwM, Exception)
+import Control.Monad.Catch (Exception)
+
+data QueryError =
+  QueryError Text
+  | InternalError Text
```

Comment:
Can we please rename this `SchemaError` (or add something to the sum type to specifically indicate errors in the definition of the schema)?